dev/com.ibm.ws.st.liberty.gradle/src/com/ibm/ws/st/liberty/gradle/internal/LibertyGradle.java

```diff
@@ -181,8 +200,8 @@ public boolean isSupportedProject(IProject project, IProgressMonitor monitor) {
     /** {@inheritDoc} */
     @Override
     public void updateSrcConfig(IPath location, LibertyBuildPluginConfiguration config, IProgressMonitor monitor) {
-        String goal = "package liberty:install-apps";
-        runGradleCommand(location, goal, config.getActiveBuildProfiles(), monitor);
+    	String[] tasks = new String[] {LibertyGradleConstants.ASSEMBLE_TASK, LibertyGradleConstants.INSTALL_APPS_TASK};
```

Comment:
Indent alignment

Reply:
I think git does not show well when there are mixed tabs and spaces as it looks fine in an editor.  I changed all tabs in the file to spaces in the file.